benchmarks/serialization/benchmarker.rb

```diff
@@ -0,0 +1,48 @@
+require 'puppet'
+
+class Benchmarker
+  def initialize(target, size)
+    @direction = ENV['SER_DIRECTION'] == 'generate' ? :generate : :parse
+    @format = ENV['SER_FORMAT'] == 'pson' ? :pson : :json
+
+    puts "Benchmarker #{@direction} #{@format}"
+  end
+
+  def setup
+  end
+
+  def generate
+    path = File.expand_path(File.join(__FILE__, '../catalog.json'))
+    puts "Using catalog #{path}"
+
+    @data = File.read(path)
+    @catalog = JSON.parse(@data)
+  end
+
+  def run(args=nil)
+    0.upto(ENV['ITERATIONS'].to_i) do |i|
```

Comment:
Good point, I'll fix that. I realized I have the wrong ticket number on some of the commits and branch name. I'm going to close this PR and reopen a new one.